ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vmtracer-sessions.md

```diff
@@ -55,8 +55,10 @@ interface Management1
 
 | Session | URL | Username | Autovlan | Source Interface |
 | ------- | --- | -------- | -------- | ---------------- |
-| session_1 |  https://192.168.0.10 | user1 | disabled |  Management1 |
-| session_2 |  https://192.168.0.10 | user1 | enabled |  - |
+|
```

Comment:
something is wrong in this output. If you check before and after as rendered markdwon.

Reply:
Corrected.